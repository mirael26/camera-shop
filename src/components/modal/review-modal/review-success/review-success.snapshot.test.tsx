import { renderWithReduxAndRouter } from '../../../../test/helpers/render-with-redux-and-router';
import ReviewSuccess from './review-success';

test('ReviewSuccess renders correctly', () => {
  const { asFragment } = renderWithReduxAndRouter(<ReviewSuccess onModalClose={() => {}}/>);
  expect(asFragment()).toMatchSnapshot();
});
