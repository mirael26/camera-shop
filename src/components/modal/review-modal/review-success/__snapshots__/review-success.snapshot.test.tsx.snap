// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReviewSuccess renders correctly 1`] = `
<DocumentFragment>
  <div
    class="modal__content"
    data-testid="review-success"
  >
    <p
      class="title title--h4"
    >
      Спасибо за отзыв
    </p>
    <svg
      aria-hidden="true"
      class="modal__icon"
      height="78"
      width="80"
    >
      <use
        xlink:href="#icon-review-success"
      />
    </svg>
    <div
      class="modal__buttons"
    >
      <button
        class="btn btn--purple modal__btn modal__btn--fit-width"
        type="button"
      >
        Вернуться к покупкам
      </button>
    </div>
    <button
      aria-label="Закрыть попап"
      class="cross-btn"
      type="button"
    >
      <svg
        aria-hidden="true"
        height="10"
        width="10"
      >
        <use
          xlink:href="#icon-close"
        />
      </svg>
    </button>
  </div>
</DocumentFragment>
`;
